lua/obsidian/commands/new_from_template.lua

```diff
@@ -0,0 +1,45 @@
+local util = require "obsidian.util"
+local log = require "obsidian.log"
+
+---@param client obsidian.Client
+return function(client, data)
+  ---@type obsidian.Note
```

Comment:
this needs to be right above `local note` (line 12).
```suggestion
  ---@type obsidian.Note
```

Reply:
Done